smoke/IotEdgeQuickstart/details/UriSocks.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+namespace IotEdgeQuickstart.Details
+{
+    internal class UriSocks
+    {
+        const int ManagementPort = 15580;
```

Comment:
where is this used?

Reply:
You're right. Removing some artifacts from the past.